backends/ebpf/runtime/ebpf_runtime.c

```diff
@@ -14,89 +14,270 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-/* The runtime emulates a loaded eBPF program and provides an interface to feed
-   packets to the emulated eBPF filter. */
+/* The runtime emulates a loaded eBPF program. It parses a set of given pcap files
```

Comment:
The runtime in fact executes a C program that processes packets.
The C program has a fixed interface: it takes an "skbuf" and returns a boolean for each packet.